### DISCUSSION TITLE
Why color space names with parentheses, e.g. "NTSC (1987)" cannot be plotted?

### DISCUSSION THREAD

**Discussion by yulincoder** [colour-science/colour#1037] [Q&A]
### Description

When I type code following:
```py
plot_RGB_chromaticities_in_chromaticity_diagram_CIE1931(
    [0.333, 0.333, 0.333],
    colourspaces=['sRGB', 'NTSC (1987)', 'Apple RGB'])
```
I can't get 'NTSC (1987)' graphics. And I traced the cause of this bug:
In the function of `colour/utilities/common.py: def filter_mapping`, colorspace name matching uses regular expressions `re.match` following：
```py
        elements = [
            mapping[element]
            for element in mapping
            if re.match(filterer, element, flags)
        ]
```
Therefore, parentheses are treated as keywords of regular expressions.

### Code for Reproduction

```python
from colour.plotting import *
plot_RGB_chromaticities_in_chromaticity_diagram_CIE1931(
    [0.333, 0.333, 0.333],
    colourspaces=['sRGB', 'NTSC (1987)', 'Apple RGB'])
```
![image](https://user-images.githubusercontent.com/10429180/186668650-b7685576-3602-42ec-96ff-c61dad523d43.png)

### Environment Information

```shell
colour-science: 0.4.1
Computer: macOS, Linux Mint
```

**Comment by thomasmansencal** [comment]
Hi @yulincoder,

This is expected, the signature of the [definition](https://colour.readthedocs.io/en/develop/generated/colour.plotting.plot_RGB_colourspaces_in_chromaticity_diagram_CIE1931.html#colour.plotting.plot_RGB_colourspaces_in_chromaticity_diagram_CIE1931) refers to the [colour.plotting.common.filter_RGB_colourspaces](https://colour.readthedocs.io/en/develop/generated/colour.plotting.common.filter_RGB_colourspaces.html#colour.plotting.common.filter_RGB_colourspaces) definition which accepts Regex patterns. It wraps itself the [colour.plotting.common.filter_passthrough](https://colour.readthedocs.io/en/develop/generated/colour.plotting.common.filter_passthrough.html#colour.plotting.common.filter_passthrough) definition that has the use case you gave above.

**Comment by yulincoder** [comment]
Ok, I see.  I can use `'NTSC \(1953\)'` to get the expected graphic. 
And whether this rule can be explained in an obvious position? such as the top-level function. It really brings a lot of trouble.

**Reply by thomasmansencal** [reply]
Next version of Colour will use a normal string so `NTSC (1953)` will work but also the new lower, slugified and canonical keys:

- `ntsc (1953)`
- `ntsc-1953`
- `ntsc1953`

Cheers,

Thomas

**Reply by yulincoder** [reply]
Thanks Thomas. 谢谢

**Comment by fredsavoir** [comment]
Thanks 🙏
